Getting Started docs broken?

> To generate the `index.html` page, type:
> 
> ```
> $ node .
> ```

```
chris@devbox ~/code/html5please (master) $ node .

module.js:340
    throw err;
          ^
Error: Cannot find module '/Users/chris.rebert/code/html5please'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:902:3
chris@devbox ~/code/html5please (master) $ node -v
v0.10.26
```

I had to instead run `node ./lib/` to get it to work.

--- a/README.md
+++ b/README.md
@@ -21,13 +21,13 @@ Make sure you have all dependencies installed.
 2. To generate the `index.html` page, type:
 
   ```sh
-  $ node .
+  $ node ./lib/
   ```
 
 3. To check if the urls are accurate (and for deploying to production), type:
 
   ```sh
-  $ node . checkurls
+  $ node ./lib/ checkurls
   ```
 
 4. All styles must be in the included Sass file rather than be updated in the generated CSS file. To compile them is to use [Compass](http://compass-style.org/):

